web/ideabag2-web/src/views/IdeasList.vue

```diff
@@ -22,11 +22,11 @@ export default {
 	components: { IdeaList },
 	activated() {
 		if (this.$store.getters.categories) {
-			var title = this.$store.getters.categories[this.$route.params.categoryId].categoryLbl;
+			const categoryId = this.$route.params.categoryId
+			var title = this.$store.getters.categories[categoryId - 1].categoryLbl;
```

Comment:
Try to use **const** or **let** so that the variable is scoped to the function instead of globally with **var**.

I always start with **const**. It prevents you from reassigning the value of the variable.
If I need to be able to reassign the value, I go from **const** to **let**.

I rarely ever use **var**, only when I need to make the variable available to the global scope.

Reply:
I too mostly use const as you can see in the commit.
The var is from your previous code.
Because the code is not so recent, you used var everywhere.

Reply:
Ooh. Sorry. Please fix it for me. I'm going to do cleanup soon (after my exams) that'd remove all the old things I've learnt better ways to do.